src/livekit/options.ts

```diff
@@ -23,10 +33,13 @@ const defaultLiveKitPublishOptions: TrackPublishDefaults = {
   red: false,
   forceStereo: false,
   simulcast: true,
-  videoSimulcastLayers: [VideoPresets.h180, VideoPresets.h360] as VideoPreset[],
+  videoSimulcastLayers: [
+    VideoPresetsH264.h144,
+    VideoPresets.h360,
+  ] as VideoPreset[],
   screenShareEncoding: ScreenSharePresets.h1080fps30.encoding,
   stopMicTrackOnMute: false,
-  videoCodec: "vp8",
+  videoCodec: "h264",
```

Comment:
```suggestion
  videoCodec: platform === "ios" || !useH264Setting ? "vp8" : "h264",
```

Adding a dev setting for using h264 would allow merging this and would provide us more data on how this behaves in real live.

Let me know if I should update this PR.

Reply:
@fkwp any thoughts on this? This would also be nice to have in the 15 release ?